---
title: Wnt ligand--receptor landscape in brachiopod axial patterning
documentclass: article
bibliography: wnt.bib 
csl: biomed-central.csl
papersize: a4
geometry:
  - bindingoffset=2mm
  - hmargin=25mm
  - vmargin=25mm
fontsize: 11pt
fontfamily: libertine
linestretch: 1
link-citations: true
colorlinks: true
fignos-warning-level: 1
fignos-cleveref: True
fignos-plus-name: Figure
tablenos-cleveref: True
tablenos-plus-name: Table
---

Bruno C. Vellutini^1,2^, José M. Martín-Durán^1,3^, Aina Børve^1,4^ and Andreas
Hejnol^1,4,5^

^1^Sars International Centre for Marine Molecular Biology, University of
Bergen, Thormøhlensgate 55, 5006 Bergen, Norway.

^2^ Max Planck Institute of Molecular Cell Biology and Genetics,
Pfotenhauerstraße 108, 01307 Dresden, Germany.

^3^ Queen Mary University of London, School of Biological and Chemical
Sciences, Mile End Road, Fogg Building, E1 4NS London, UK

^4^Department of Biological Sciences, Molecular Biology, University of Bergen,
Thormøhlensgate 55, 5006 Bergen, Norway.

^5^ Institute of Zoology, Friedrich Schiller University Jena, Erbertstraße 1,
07743 Jena, Germany

<!--Evolutionary Research and Phyletic Museum-->

<!--Outline:

#title: Wnt signaling landscape in brachiopod anteroposterior patterning
#title: Involvement of Wnt signaling ligand-receptor context in brachiopod anteroposterior patterning
#title: Involvement of Wnt signaling in the embryonic patterning of brachiopod body regions
#title: Expression of Wnt signaling components suggests that ligand-receptor context plays a role in the patterning of different brachiopod body regions
#title: Expression of Wnt signaling components correlates with the anteroposterior patterning of the brachiopod *Terebratalia transversa*
#title: Wnt signaling ligand-receptor landscape during brachiopod anteroposterior patterning
#title: Wnt signaling unique ligand-receptor contexts demarcate different body regions during brachiopod development 
#title: Wnt signaling in the brachiopod *Terebratalia transversa*
#title: Wnt signaling combinatorial landscape patterning the anteroposterior axis of brachiopod embryos
#title: Wnt signaling ligand-receptor context during brachiopod anteroposterior patterning
#title: Wnt signaling ligand-receptor context during brachiopod embryonic development
#title: Brachiopod body regions express unique ligand-receptor contexts of Wnt signaling during embryonic development
#title: Brachiopod body regions are patterned by unique ligand-receptor context of Wnt signaling
#title: Unique Wnt signaling ligand-receptor context demarcates different body regions during brachiopod development 
#title: Unique Wnt signaling ligand-receptor contexts demarcate different brachiopod body regions
#title: Unique ligand-receptor Wnt subregions patterning the anteroposterior axis of brachiopod embryos
#title: Unique combinations of Wnt-Frizzled are expressed along the anteroposterior axis of the brachiopod *Terebratalia transversa*
#title: Modular Wnt signaling ligand-receptor contexts along the anteroposterior axis  during brachiopod development
#title: Patterning of brachiopod anteroposterior axis by combinatorial Wnt signaling landscape
#title: Combinatorial expression of Wnt signaling components demarcate brachiopod body regions during embryonic development
#title: Combinatorial ligand-receptor expression of Wnt components in brachiopod embryo suggests a an role in anteroposterior patterning  
#title: Combinatorial landscape of Wnt-Frizzled expression domains along the anteroposterior axis of brachiopod embryos
#title: Combinatorial landscape of Wnt-Frizzled expression domains suggests a role of Wnt signaling in the anteroposterior patterning of brachiopod embryos
#title: Combinatorial expression of ligand-receptor suggests an axial patterning role for Wnt signaling in brachiopod embryos
#title: Ligand-Receptor map reveals Wnt signaling landscape patterning the anteroposterior axis in the brachiopod *Terebratalia transversa*
#title: Ligand-receptor context of Wnt signaling during brachiopod anteroposterior patterning
#title: Wnt-Frizzled combinatorial expression domains pattern the anteroposterior axis in the brachiopod *Terebratalia transversa*

    # Background

    - Wnt signaling is crucial for embryonic development in multiple ways
    - One major role is anteroposterior patterning and developmental boundaries
    - Receptor context might be crucial for signaling outcome
    - Brachiopods are important spiralian evo-devo organisms
    - Larval body is subdivided into lobes that involve Wnt signaling
    - Investigate Wnt signaling role in patterning body regions
    - Find Wnt/Frizzled expression coincides with body regions and boundaries
    - Each subregion has a unique ligand-receptor context
    - Receptors are more conserved while ligands more labile
    - Some ligand-receptor combinations are conserved in other animals
    - Suggests that Wnt play a role in patterning larval body regions
    - And that receptor context might matter for signaling outcome

    # Results by gene family

    - Wnt ligands
    - Frizzled receptors
    - Wnt antagonists
    - Intracellular components
    - Cell polarity components
    - Wnt signaling landscape

    # Results by experiment and body region

    - Wnt complement
    - Expression profiling
    - Expression domains Inhibitor-Ligand-Receptor
      - Anterior apical lobe: dkk, sfrp, fz1/2/7, fz5/8
      - Posterior apical lobe: fz1/2/7, fz4, wnt2, wnt8, wnt5, wntA
      - Apical-mantle boundary: fz1/2/7, fz9/10, wnt1, wnt7
      - Mantle lobe: fz1/2/7, wnt5, wnt16
      - Anterior pedicle lobe: fz9/10, wnt1, wnt4, wnt9, wnt10
      - Posterior pedicle lobe: wnt5, wnt6, wnt11
    - Expression domains intracellular
    - Expression domains cell polarity

    # Results by key messages

    - Wnt complement is mostly complete
    - Few Wnt ligands are maternally deposited in the egg
    - Wnt ligands are progressively upregulated in two waves of activation at
      the blastula and gastrula stages
    - Every Wnt ligand is expressed in partially overlapping domains in the
      late gastrula stage at the onset of body specification
    - Expression of some Wnt ligands fade after metamorphosis


    # Discussion

    - Wnt complement is mostly complete
    - Maternal ligands are variable, receptors conserved
    - Molecular territories match AP regions, suggests patterning
    - Common receptor map, highly modular ligand deployment
    - Revisiting the Wnt code to uncover morpho and evo

-->

# Abstract

Wnt signaling is crucial for the anteroposterior patterning of bilaterians. It
has a role in axial determination, fate specification and other key embryonic
events. The signaling outcome is highly complex and dependent on the local
receptor-context for Wnt ligand in the tissue. While the expression of Wnt
ligands has been analyzed in diverse taxa, the combined analysis of Wnt ligands
and Frizzled receptors is often lacking. Here, we analyze the developmental
expression of Wnt components in the brachiopod *Terebratalia transversa*, a
spiralian whose larval body is subdivided by distinct morphological boundaries.
We find that Frizzled receptors are expressed in broad domains while Wnt
ligands are expressed in localized overlapping territories along the
anteroposterior axis. We identify a series of transcriptional subregion, with
Wnt antagonists at the anterior end, and a receptor-free zone at the posterior
end, and each subregion expresses a unique ligand--receptor combination which
coincides with the morphological boundaries that divide the brachiopod body.
Our expression data suggests that the ligand--receptor context plays a key role
in the differential patterning, and subsequent morphogenesis of the larval body
regions of the brachiopod *T. transversa*, and other animals. This suggests
that the shuffling of ligand--receptor contexts, might be important to the
evolution of bilaterian body diversity.

# Background

<!--TODO: Draft paragraphs for introduction-->

Wnt signaling is crucial for embryonic development in multiple ways. It is
involved in fate specification, organogenesis, etc. There is an early role in
axis determination. But also a major role is anteroposterior patterning of the
body regions.

It has been increasingly recognized that receptor context is crucial for
signaling outcome. That multiple ligand receptor combinations might trigger
different pathways and differentiation. For example, ... . Thus understanding
the receptor context for the Wnt expression is important.

Most Wnt work has been focused in deuterostomes and arthropods. The Spiralia is
another major bilaterian clade with great diversity and crucial to understand
bilaterian evolution.

Brachiopods are important spiralian evo-devo organisms. There is substantial
work on gene expression Hox, segmentation, hedgehog, gastrulation, etc. The
larval body is typically subdivided into lobes that involve Wnt signaling.

The goal of this study is to investigate if Wnt signaling has a role in
patterning the different body regions using T. transversa. We analyzed
stage-specific RNA-seq data and spatiotemporal expression of Wnt signaling
components.

We find that Frizzled expression occurs in broad domains that cover almost the
whole brachiopod extension. Wnt ligands are expressed in subregions, stripes or
patches, often partially overlapping.

Each subregion has a unique ligand-receptor context. Antagonists are restricted
to the anterior region and receptor is cleared from the posterior region.
Ligand receptor coincides with body regions and morphological boundaries.
Suggests that Wnt play a role in patterning brachiopod larval body regions.
Some ligand-receptor combinations are conserved in other animals. Receptors are
more conserved while ligands more labile. Indicates, that receptor context
shuffling matter for bilaterian evolution.

<!--At the Fz receptor, the binding of proteins other than Wnt can antagonize signaling. Specific antagonists include Dickkopf (Dkk), Wnt inhibitory factor 1 (WIF-1),[50][51] secreted Frizzled-related proteins (SFRP), Cerberus, Frzb, Wise, SOST, and Naked cuticle. These constitute inhibitors of Wnt signaling. However, other molecules also act as activators. Norrin and R-Spondin2 activate Wnt signaling in the absence of Wnt ligand.-->

<!--Wnt signaling pathway plays a major role in animal development. The ligands are secreted glycoproteins and receptors are transmembrane proteins. Early role in AV patterning and a later role in AP patterning. Some Wnt ligands are contiguous in the genome. Expression in spiralians is still understudied. Brachiopod preliminary genome, transcriptome and in situ expression data. -->

<!--TODO: The paragraphs below are more for the introduction.-->

<!--The discovery of multiple wnt genes, their crucial developmental roles and their complex expression domains has raised ideas about a wnt code [@Guder2006-fu]. An idea that different wnts can pattern and specify different body regions according to their combinatorial territories. Much in parallel with the discovery of the Hox system. However, accumulated evidence reveals that the Wnt system is much more imbricated modular and multifunctional than the Hox patterning.-->

<!--First is the finding that wnt ligands are not long- but short-ranged signaling activity [@Loh2016-pz]. The consequence is that wnt function is more local and allow for more complex patterning outcomes.-->

<!--Second, the idea that is not a specific wnt ligand that mediate specific signaling, but that the receptor context dictates the Wnt signaling outcome [@Loh2016-pz]. This implies that the same ligand can trigger different outcomes based on the receptor it binds to. This put a greater importance when studying wnt signaling on analyzing the receptor context in conjunction to ligands.-->

# Results

## Wnt ligands

<!--Wnt complement-->

Wnt ligands are secreted glycoproteins with a conserved series of cysteine
residues positioned along the coding sequence [@Nusse1992-oi]. They evolved at
the root of metazoans from thirteen ancestral subfamilies [@Kusserow2005-xd;
@Cho2010-eb]. We identified twelve of these Wnt subfamilies in the brachiopod
*T. transversa*. The only missing is *wnt3*. Most other subfamilies are
represented by a single ortholog with the exception of *wnt1*, which is
duplicated (@fig:wnt-tree). One of the *wnt1* paralogs in *T. transversa*
(*wnt1b*) shows a highly divergent sequence (@fig:wnt1-tree).

<!--Wnt profiling-->

To characterize the expression dynamics of Wnt signaling components during *T.
transversa* embryonic development, we analyzed the stage-specific RNA-seq
dataset (ACCESSION) covering representative developmental stages from
unfertilized oocytes, cleavage, and gastrulation, to larval morphogenesis and
post-metamorphosis juveniles.

We detect high levels of *wnt4* and *wntA* transcripts maternally loaded in the
unfertilized oocyte (@fig:profiling). Other Wnt genes are expressed, such as
*wnt1b*, *wnt5*, *wnt6*, and *wnt16*, but at lower levels. The remaining Wnt
ligands are barely detected at this stage.

The Wnt landscape changes at the late blastula stage (19h) when there is a
concerted upregulation of *wnt1*, *wnt8*, *wnt10* and *wnt16* in the embryo
(@fig:profiling). *wnt8* and *wnt6* are the genes that undergo the greatest
shift in expression levels. There is a second wave of activation at the onset
of gastrulation with the increase of *wnt1*, *wnt5* and *wnt6* at the early
gastrula. These transcriptional changes are followed by the upregulation of
*wnt7* and *wnt11* at the mid gastrula, and finally, by the activation of
*wnt2*, *wnt9* and *wnt10* at the late gastrula. Thus, different Wnt ligands
are upregulated at different moments during embryogenesis, but between the late
gastrula and early larva stages, all Wnts are expressed in the tissues of the
developing *T. transversa* embryo.

While the expression of *wnt1*, *wnt4*, *wnt5*, *wnt8*, *wnt9*, *wntA* remains
high until the juvenile stages, the expression levels of *wnt6* and *wnt10*
begin to decay after gastrulation. After metamorphosis, we detect a notable
downregulation of *wnt7* and *wnt16* in the juvenile individuals
(@fig:profiling).

<!--Wnt expression-->

To uncover the spatial localization of Wnt genes in developing embryos of *T.
transversa*, we analyzed the expression of the thirteen Wnt ligands between the
early gastrula and the late larva stage by *in situ* hybridization
(@fig:ligands1 and !@fig:ligands2).

*wnt1* expression has been previously described [@Vellutini2016-lq]. In early
and mid gastrula stages, it is expressed in a terminal position at the
posterior blastopore lip (@fig:ligands1 and @Vellutini2016-lq). In late
gastrula and early larva, this posterior domain acquires a subterminal position
forming bilateral bands in the pedicle lobe, while a novel domain, a pair of
narrow *wnt1* stripes appear in the apical lobe (@fig:ligands1 and
@Vellutini2016-lq). In late larva, the posterior *wnt1* domain forms a ring
subdividing the pedicle lobe, and a novel dorsal patch localized to the future
shell expands (@fig:ligands1 and @Vellutini2016-lq).

*wnt1b* is expressed in a small, localized dorsal domain in the early, mid and
late gastrula stages (@fig:ligands1). This apical domain becomes fragmented in
the early larva and disappears in the late larva. In the early larva *wnt1b* is
upregulated in a central ventral patch posterior to the mouth. In the late
larva two novel domains appear below the mantle lobe and at the terminal
posterior tip of the pedicle lobe (@fig:ligands1).

*wnt2* transcripts are not detected by in situ hybridization in the early and
mid gastrula stages. At the late gastrula, broad bilateral bands of
*wnt2*-expressing cells encircle the posterior portion of the apical lobe but
without reaching the ventral or dorsal midline (@fig:ligands1). The pattern
persists at the early larva, but is cleared in the late larval stages
(@fig:ligands1).

*wnt4* is expressed at the posterior end in the early and mid gastrula stages
(@fig:ligands1) but the transcripts are localized more dorsally than *wnt1*
({@fig:doubles}A). In the early larva the domain shifts to a subterminal
position at the pedicle lobe that partially overlaps with the posterior
territory of *wnt1* ({@fig:doubles}B). In the late larva, *wnt4* is expressed
in a ring at the anteriormost portion of the pedicle
lobe (@fig:ligands1).

*wnt5* exhibits three distinct ectodermal domains of expression in *T.
transversa*. They localize to the apical, mantle and pedicle lobe of the larva
(@fig:ligands1). In the early gastrula we detect *wnt5* transcripts at the
posterior end of the blastopore lip (@fig:ligands1). During gastrulation this
domain becomes narrower than the overlapping posterior domain of *wnt1*
({@fig:doubles}C) acquires a terminal position in the developing pedicle lobe
(@fig:ligands1 and {@fig:doubles}D,E). In the late larva stage, with the
subdivision of the pedicle lobe, *wnt5* is cleared from its terminal portion
but remains strongly expressed in a subterminal region of the pedicle lobe
(@fig:ligands1). The other two domains of *wnt5* are also detected early during
gastrulation and form broad bilateral territories that correspond to the middle
region of the apical and mantle lobes (@fig:ligands1). Except for a narrow row
of cells at the apical lobe, and at the posterior blastopore lip, the
expression domains of *wnt5* and *wnt1* do not coincide ({@fig:doubles}D,E). In
the early and late larva stages the apical domain of *wnt5* becomes weaker,
while the mantle lobe territory becomes restricted to the growing edge of the
mantle lobe (@fig:ligands1).

*wnt6* is expressed at the posterior blastopore lip in the early and mid
gastrula stages. At the end of gastrulation *wnt6* is also expressed at the
middle portion of the closing blastopore lip in a domain that localizes to
endodermal tissues (@fig:ligands1). At the late larva *wnt6* is faintly
expressed in the apical, mid gut and pedicle lobe.

*wnt7* transcripts are detected early during gastrulation in a bilateral pair
of laterodorsal patches (@fig:ligands1). These patches develop into a thin
stripe of expression localizing precisely to the boundary between the apical
and mantle lobes (@fig:ligands1). The expression partially overlaps with the
*engrailed*-positive stripe of cells at the anteriormost region of the mantle
lobe ({@fig:doubles}F; see also [@Vellutini2016-lq]). One additional striped
domain of *wnt7* appears in the pedicle lobe of the early larva at the
interface between the *wnt1* and *wnt5* posterior territories
({@fig:doubles}F).

*wnt8* is highly expressed in the early gastrula stage in a posterodorsal
domain and in a pair of anterolateral territories (@fig:ligands2). While the
posterior domain fades during gastrulation, the anterior transcripts expand
ventrally and dorsally forming broad bilateral bands of *wnt8* expression in
the apical lobe. This domain partially overlaps with the anterior *wnt1*
transcripts ({@fig:doubles}G).

*wnt9* transcripts are detected in the late gastrula stage on the middle
portion of the blastopore lip. These cells are internalized and, in the late
larva stage, *wnt9* is expressed in a pair of anterior bilateral mesodermal
domains and in the endoderm (@fig:ligands2).

*wnt10* expression begins in the posterior end at the mid gastrula stage in a
domain that localizes to the pedicle lobe in the larva (@fig:ligands2).

*wnt11* is strongly expressed in a posterodorsal domain during gastrulation. At
the early and late larva stages *wnt11* expressing cells encircle a subregion of
the pedicle lobe (@fig:ligands2). In addition, we detect *wnt11* transcripts
in more anterior patches in the pedicle lobe during gastrulation and in the mid
and posterior gut in the early larva stage.

*wnt16* transcripts are detected in broad bilateral domains around the
blastopore opening in the early gastrula (@fig:ligands2). The domains localize
to the mantle lobe, are internalized during gastrulation and in later stages
*wnt16* expression is present in an anterior mesodermal domain.

*wntA* is not expressed during early stages of gastrulation but in a pair of
broad ventral patches bordering the blastopore in the apical lobe in late
gastrula and early larva stages (@fig:ligands2). At the late larva *wntA* is
expressed in a pair of mesodermal bands that extend adjacent to the mouth
opening lining the anterior and mid endoderm.

<!--TODO: Previous results. Something to use?-->

<!--*T. transversa* Wnt ligands are expressed in diverse ectodermal domains in all body regions except the apical portion of the apical lobe. The anteriormost Wnt transcripts are bilateral anterior territories of *wnt5* and *wnt8* that we detect in the early and mid gastrula stages. At the same stage we also observe lateral patches of *wnt16* next to the blastopore. Many Wnt genes are expressed at the posterior portion of the blastoporal lip, such as *wnt1*, *wnt6* and *wnt11*, but Wnt ligands are notably absent from the anterior portion of the blastoporal lip. In addition, Wnt ligands are also expressed in the mantle lobe (e.g., *wnt5*), the middle section of the larval body, and in more localized endodermal and mesodermal tissues (e.g. *wnt6* and *wnt9*).-->

<!--In summary, the expression territories of Wnt ligands in *T. transversa* partially overlap and can be divided into four groups of genes. Anterior domains in the apical lobe (*wnt2*, *wnt8*, *wnt5*, *wntA*), mid body domains in the mantle lobe (mainly *wnt5* and *wnt16*), subterminal domains in the pedicle lobe (*wnt1*, *wnt4*, *wnt9* and *wnt10*), and terminal domains in the pedicle lobe (*wnt5*, *wnt6*, *wnt11*).-->

<!--The early expression domains at the anterior and posterior regions initially overlap. However, as embryonic development progresses, these territories differentiate into distinct subregions of the larval body. For instance, the overlapping domains at the posterior portion of the blastopore will segregate and occupy different subregions of the pedicle lobe in the late larva stage. -->

<!--In addition, *wnt1* and *wnt7* have domains of expression at the boundaries between the apical and mantle lobes, and between the subterminal and terminal portions of the pedicle lobe. These Wnt domains differentially overlap with the receptor territories forming a complex landscape of ligand-receptor combinations.-->

## Frizzled receptors

<!--Frizzled orthology-->

Frizzled receptors are transmembrane proteins with an extracellular
cystein-rich domain and seven hydrophobic transmembrane domains
[@Huang2004-di]. Metazoans show a variety of Frizzled proteins which can be
divided into four subfamilies: *fz1/2/7*, *fz3/6*, *fz5/8*, *fz9/10*, and *fz4*
[@MacDonald2012-ri]. In *T. transversa*, we identified four Frizzled proteins,
a single ortholog for each subfamily with the exception of *fz3/6*
(@fig:fzd-tree).

<!--Frizzled profiling-->

Transcripts of *fz1/2/7*, *fz4* and *fz5/8* are maternally deposited and highly
expressed until the late blastula stage while *fz9/10* expression increases at
the onset of gastrulation (@fig:profiling). The levels of *fz1/2/7* and *fz4*
remain high while *fz9/10* and *fz5/8* expression decrease in the juvenile
stages (@fig:profiling).

<!--Frizzled expression-->

*fz1/2/7* is expressed in the ectoderm and mesoderm in the early gastrula
(@fig:receptors). During gastrulation the transcripts are slightly cleared from
the pedicle lobe and *fz1/2/7* remains expressed in anterior ecto and
mesodermal tissues. In the late larva stage, we also detect transcripts at the
terminal portion of the pedicle lobe (@fig:receptors).

*fz4* is expressed in a subapical ectodermal ring encircling the animal pole in
the early gastrula stage and in the anterior portion of the mesoderm
(@fig:receptors and @fig:supplementary). The *fz4* ring demarcates the base of
the apical lobe and, together with the anterior mesoderm domain, is expressed
throughout development. In the larval stages we detect novel ectodermal domains
of *fz4* at the boundary between the mantle and pedicle lobe and at the
posterior tip of the pedicle lobe (@fig:receptors).

*fz5/8* transcripts form an ectodermal apical domain in the early gastrula
stage (@fig:receptors and @fig:supplementary). This domain is complementary to
the *fz4* domain in the apical lobe. During gastrulation *fz5/8* begins to be
expressed in the anterior portion of the mesoderm in a domain narrower than the
*fz4* territory (@fig:receptors). In the late larva stage both anterior ecto
and mesodermal domains continue to be expressed, and *fz5/8* transcripts are
also detected in the mesoderm of chaetae sacs (@fig:receptors).

*fz9/10* is expressed in a broad ectodermal territory around the blastopore lip
in the early gastrula stage (@fig:receptors). Transcripts are progressively
cleared first from the apical domain, then from the posterior portion of the
embryo and *fz9/10* expression remains mostly in the middle portion of the body,
which extend from the posterior portion of the apical lobe to the anterior
portion of the pedicle lobe (@fig:receptors). *fz9/10* is broadly expressed in
the mesoderm throughout development and is also detected in the endoderm of the
early larva stage (@fig:receptors).

<!--TODO: Previous results. Something to use?-->

<!--Frizzled receptors are expressed in complex spatial patterns that include broad non-overlapping ectodermal domains. However, they are also detected in specific mesodermal and endodermal tissues in the developing *T. transversa* embryos. Except for *fz1/2/7*, which is expressed almost ubiquitously, *fz4* and *fz5/8* are expressed in anterior subregions of the apical lobe and *fz9/10* is predominantly more posterior in the mantle and pedicle lobes (@fig:receptors).-->

<!--In summary, Frizzled receptors are expressed along the anteroposterior axis of the gastrulating embryo except for the posterior terminal portion of the pedicle lobe. These domains can be divided into five unique territories formed by a distinct combination of each receptor. At the anterior most portion of the apical lobe (*fz1/2/7* and *fz5/8*), at the medial portion of the apical lobe (*fz1/2/7* and *fz4*), at the posteriormost portion of the apical lobe and anterior most portion of the mantle lobe (*fz1/2/7* and *fz9/10*), the middle and posterior portion of the mantle lobe (*fz1/2/7*), and at the anterior portion of the pedicle lobe (*fz9/10*). The terminal region is free of Frizzled expression during gastrulation and only *fz4* and *fz1/2/7* are upregulated in the late larva stage.-->

## Wnt antagonists

<!--Antagonist orthology-->

The secreted frizzled-related proteins (*sfrp*) are soluble proteins that
antagonize Wnt activity by direct binding to Wnt ligands or to Frizzled
receptors [@Kawano2003-bx]. They have a cystein-rich domain with high-affinity
to Wnt proteins. The *sfrp* family can be divided into two subfamilies,
*sfrp1/2/5* and *sfrp3/4* [@Kawano2003-bx; @Bovolenta2008-kb]. In *T.
transversa* transcriptome, we only identified a *sfrp1/2/5* ortholog
(@fig:sfrp-tree). *sfrp1/2/5* is highly expressed throughout development
(@fig:profiling). It localizes to an apical ectodermal domain and to a narrow
anterior mesoderm patch in a pattern that recapitulates the expression of
*fz5/8* (@fig:antagonists). In the early larva, we detect an additional
bilateral domain of *sfrp1/2/5* expression in the mesoderm of the mantle lobe
(@fig:antagonists). 

The *dickkopf* (*dkk*) are secreted glycoproteins that antagonize Wnt signaling
by inhibiting the *lrp5/6* co-receptors [@Niehrs2006-kf; @Cruciat2013-zs]. They
contain two cystein-rich domains which can be divided into two subfamilies,
*dkk1/2/4* and *dkk3* [@Niehrs2006-kf]. In *T. transversa*, we identified a
single *dkk* ortholog. It groups with a previously unidentified *dkk* subfamily
that differs from the dkk1/2/4 and dkk3 subfamilies, and named hereafter as
*dkk5* (@fig:dkk-tree). The expression of *T. transversa* *dkk5* is upregulated
at the late blastula stage and downregulated in the juvenile stages
(@fig:profiling). Spatially, the expression is confined to an apical ectodermal
domains, similar to *sfrp1/2/5* (@fig:antagonists).

The wnt inhibitory factor (*wif*) is another protein that inhibits Wnt activity
by direct binding to Wnt proteins [@Hsieh1999-rt]. The protein has five EGF
repeats and a typical unique *wif* domain which is shared with RYK receptor
tyrosine kinases [@Hsieh1999-rt; @Cruciat2013-zs]. In *T. transversa* we
identified one *wif* gene (@fig:wif-tree). It has a relatively low and stable
expression levels throughout development with a peak at the late gastrula stage
(@fig:profiling). Unlike *sfrp1/2/5* and *dkk*, *wif* is mainly expressed in
the mesoderm from gastrulation to the late larva stage (@fig:antagonists).

## Intracellular components

<!--Components orthology and profiling-->

The gene dishevelled (*dsh*) is a central regulator of Wnt signaling pathways
[@Wallingford2005-xp]. The protein has three conserved domains (DIX, PDZ, and
DEP domains), and two conserved regions before and after the PDZ domain (a
cluster of basic residues and a proline-rich region, respectively)
[@Gao2010-wj]. The DIX domain is shared with *axin*, a negative regulator of
Wnt signaling that interacts directly with *dsh* [@Li1999-sy]. The interaction
of *dsh* and *axin* controls the degradation of *β-catenin*, the downstream
transcriptional activator of the canonical pathway [@Miller1996-rr]. In *T.
transversa*, we identified single copies of *dsh*, *axin*, and *β-catenin*
orthologs (@fig:dsh-tree and @fig:bcat-tree). The three genes are
highly-expressed in every developmental stage. *dsh* and *axin* levels decrease
after the mid blastula stage, while the abundance of *β-catenin* transcripts
remains stable throughout development (@fig:s-profiling).

<!--Components expression-->

*dsh* transcripts localize to a narrow apical domain in the ectoderm, and to
adjacent territory in the anterior mesoderm (@fig:intracellular). The apical
domain is cleared in the late larva and *dsh* is upregulated in the dorsal
mesoderm of the mantle lobe.

*axin* is broadly expressed in the early gastrula with higher levels at the
posterior end (@fig:intracellular). At the late gastrula and early larva it is
mostly expressed in a pair of bilateral bands at the anterior portion of the
pedicle lobe which correspond to the *wnt1* domain, as well as in the mantle
lobe mesoderm adjacent to the endoderm (@fig:intracellular).

*β-catenin* is ubiquitously expressed in the ectoderm and mesoderm until the
late gastrula when the ectodermal transcripts are downregulated
(@fig:intracellular). During larval development, mesoderm expression is also
downregulated. In the late larva stage, *β-catenin* transcripts are
restricted to the anterior mesoderm and to the mesoderm associated with the
chaetae sacs in the mantle lobe (@fig:intracellular).

## Cell polarity components

<!--TODO: Integrate the pcp pathway into the manuscript.-->

We identified *T. transversa* orthologs for components of the cell
polarity pathway (PCP Pathway, non-canonical Wnt Pathway) and analyzed their
expression during the period of axial elongation (early to late gastrula).

*diego* is a cytoplasmic protein that suppresses the cell fate pathway and
stimulates the cell polarity pathway [Schwarz-Romond2002-td]. It contains six
ankyrin repeat domains [@Feiguin2001-zd] while its vertebrate counterpart
*diversin* now known as *ankrd6* has eight [Schwarz-Romond2002-td]. We
identified a *dgo* ortholog in T. transversa containing six ankyrin repeats
(@fig:dgo-tree). *dgo* is only expressed in the late gastrula stage in
bilateral laterodorsal domains at the apical lobe and at the pedicle lobe
(@fig:s-polarity).

*prickle* competes with *diego* for *dishevelled* binding [@Jenny2005-vf]. The
protein contains a PET domain and three LIM domains [@Gubb1999-ba]. In *T.
transversa* we identified a single *pk* ortholog (@fig:pk-tree). *pk* is
expressed as a posterior patch in the early gastrula stage (@fig:s-polarity).
In the late gastrula it is expressed in ventral bilateral patches in the apical
lobe complementary to the areas absent of *jnk* transcripts. *prickle* is also
upregulated in the mesodermal pouches (@fig:s-polarity).

*flamingo* is a seven-pass transmembrane cadherin that regulates cell polarity
[@Usui1999-fv; @Chae1999-ci]. In *T. transversa* we identified one ortholog of
fmi (@fig:fmi-tree). Transcripts are present throughout development in most
embryonic tissues. Bilateral patches in the apical lobe become stronger in the
late gastrula (@fig:s-polarity).

*strabismus* is expressed ubiquitously throughout development (@fig:s-polarity).

*jnk* is faintly expressed in the early gastrula. In the mid gastrula it is
present in the archenteron posterior portion. In the late gastrula jnk
expression is upregulated almost ubiquitously across the embryo, with the
exception of ventral lateral regions in the apical lobe (@fig:s-polarity).

# Discussion

## *T. transversa* shows a conserved ancestral Wnt complement 

Metazoans have an ancient repertoire of 13 Wnt subfamilies [@Kusserow2005-xd;
@Croce2009-vv; @Cho2010-eb]. Our data suggests the brachiopod *T. transversa*
has a conserved set of Wnt genes. Only one Wnt is missing---*wnt3*, which has
been lost at the base of the protostome lineage [@Cho2010-eb;
@Janssen2010-dc]---and only *wnt1b* has duplicated. Duplications in Wnt genes
are common. The brachiopod *Lingula anatina* duplicated *wnt1* and *wnt10*
[@Luo2015-lm]. Our phylogenetic analysis suggests the duplication of *wnt1* in
*T. transversa* and *L. anatina* occurred independently. The sequence of *T.
transversa* *wnt1b* is highly divergent compared to other protostome orthologs,
suggesting a rapid evolution and supporting the hypothesis of a
lineage-specific duplication. The unusual expression of *wnt1b* compared to the
other ligands suggests that it might have evolved novel functions, or different
regulatory mechanisms.

<!--Frizzled is missing fz3/6 which is vertebrate-specific. Single-copy of sfrp and wif inhibitors.-->

dkk is interesting. T.transversa dkk3 does not group with the cnidarian dkk3 or
human dkk3, but in a distinct group of mainly protostomes. While hemichordates
have the two subfamilies of dkk3. This suggests there are two distinct families
of dkk3. Structurally their domains differ. The ancestral dkk3 (no-wnt) gave
rise to dkk1/dkk3protostome (yes-wnt). dkk3 is seemingly unrelated to Wnt
signaling. But dkk3 has been shown to modulate wnt in one amphioxus species but
another. Our phylogenetic analysis suggests that the three subfamilies of dkk
were present in hemichordates and amphioxus. While wntdkk3 was lost in
deuterostomes, dkk1 and nowntdkk3 was lost in protostomes. In summary, dkk
found in protostomes is closer to dkk1-2-4 than dkk3. which is divergent and
seemingly unrelated to Wnt signaling

## Maternal load conserved Frizzled receptor and diverse load of Wnt ligands

The early developmental role of Wnt signaling in specifying the embryonic
primary axis has been shown to occur in different bilaterian lineages [@refs].
However, in most cases, the identity of the ligand/receptor has not been
demonstrated [@refs]. Nevertheless, Wnt signaling activity in early development
depends upon the maternal load of pathway components deposited in the egg.

We found that in the oocytes of *T. transversa*, *wnt4* and *wntA* mRNAs are
the highest expressed Wnt ligands maternally loaded, suggesting they might be
involved in the early signaling events that pattern the animal-vegetal axis.
However, these specific ligands are not always deposited maternally in other
bilaterians. While most animal lineages exhibit a maternal load of Wnt ligands
(@tbl:maternal), there is considerable variability in which ligands are
deposited in the eggs. Even within brachiopods there are significant
differences in the maternal load of Wnt ligands (@fig:supplementary). This
suggests that the identity of Wnt ligands deposited in the eggs of different
bilaterians is highly labile and even vary among closely related groups with
similar developmental modes.

In contrast, most species investigated so far load in their eggs the Frizzled
receptor *fz1/2/7* (@tbl:maternal). This pattern is highly conserved among
metazoans [@Bastin2015-oz]. The contrasting variability in the ligand identity
and conservation of receptor identity suggests that the early role is deployed
regardless of the specific ligand deposited in the egg. Supports the idea that
the receptor context rather than a specific wnt ligand might crucial for the
signaling output [@Loh2016-pz].


## Unique transcriptional subregions pattern the brachiopod larval body

<!--Ligand--receptor context coincide with brachiopod body regions-->

During *T. transversa* embryonic development, Wnt ligands and Frizzled
receptors are expressed in a complex landscape of domains along the
anteroposterior axis (@fig:summary). By analyzing the combinatorial expression
of Wnt ligands, receptors and antagonists, we can identify six broad ectodermal
territories, each with a unique transcriptional profile (@tbl:landscape).
Frizzled receptors are expressed in broad domains while Wnt ligands are
expressed in partially overlapping domains with differing anteroposterior or
dorsoventral limits. Interestingly, these domains correspond to morphological
landmarks along *T. transversa* larval body.

<!--Our gene expression data reveals a complex landscape of Wnt signaling components along the anteroposterior axis of the brachiopod *T. transversa* (@fig:summary). These include six broad ectodermal territories with unique combinations of Wnt ligands and Frizzled receptors.-->

At the anteriormost tip, we observe a ligand-free zone where antagonists are
expressed. At the posteriormost tip we find a receptor-free zone where only
some ligands are expressed. The extent of the molecular territories in between
correlate with the body regions of the brachiopod larva, the apical, mantle and
pedicle lobes, and the morphological boundaries that separate them.

<!--TODO: Check role of sfrp and dkk in setting up neural fates.-->

The anterior tip expresses *fz1/2/7* and *fz5/8*. There are no Wnt ligands
expressed near the anterior tip, and given that Wnt ligands likely have a
limited diffusion [@refs], it is unlikely that the Frizzled receptors at the
anterior most tip interact with any Wnt ligand during the embryonic development
of *T. transversa*. In this region, however, both *sfrp1/2/5* and *dkk* are
expressed at the tip of the apical lobe. This inhibitors have been implicated
in determining neural fates [@refs] and are likely involved in the patterning
and formation of the neuropile and apical organ which develop in the region
[@refs]. 

<!--TODO: Do Frizzled receptors interact with something else?-->

<!--TODO: Read about interactions of these genes with fz1/2/7 and fz4.-->

Another transcriptional subregion is an anterior group with *wntA*, *wnt8*,
*wnt2* and *wnt5* that are expressed in the apical lobe with partially
overlapping domains. For instance, *wntA* is located only ventrally near the
mouth while *wnt8* is expressed laterally and *wnt2* more dorsally. While
*wnt5* encircles the whole circumference of the apical lobe. These genes are
often associated with head development [@refs]. In *T. transversa*, they are
likely involved in the subdivision of the apical lobe into a posterior enlarged
region and a more narrow anterior part [@refs].

<!--TODO: Check roles of the above wnts related to head development.-->

<!--TODO: Check morphological description of the apical lobe morphology.-->

<!--TODO: Anything known about the interactions of wnt1 with these fzs?-->

At the interface of the apical-mantle boundary there is another transcriptional
subregion which expresses a narrow ring of *wnt7*-positive cells. wnt7 has been
described to be involved in [@refs]. In addition, the apical-mantle boundary is
abutted by *wnt1* (anterior) and *engrailed* (posterior) stripes
[@Vellutini2016-lq]. This arrangement is highly conserved in arthropod trunk
segmentation and has a role in setting up segment polarity [@refs]. In
Drosophila and others, it is known or suggested that the receptor mediating
wnt1 signaling is fzX [@refs]. In *T. transversa* this seems to be the same or
different because the Frizzled receptors expressed in the apical-mantle
boundary are fz1/2/7 and fz9/10.

<!--TODO: Fetch references to support the above paragraph.-->

<!--TODO: Elaborate the wnt5 discussion, it’s interesting.-->

The mantle lobe is the region that undergoes dramatic morphogenetic changes
during development, where the tissues grow and elongate forming the mantle.
Here we identified another ligand-receptor combination. The region is covered
by *fz1/2/7* and the main ligand is *wnt5*. *wnt16* is also expressed but
restricted to the ventral portion bordering the blastopore. Interestingly,
*wnt5* regulates tissue elongation by controlling convergent extension in other
animals [see refs in @Loh2016-pz]. These morphogenetic changes might depend
upon the activation of the cell polarity cascade (noncanonical Wnt pathway).
Because the presence of *fz4* causes the activation of the *β-catenin* cascade
(canonical Wnt pathway) [@refs], the cascade signaling in the mantle lobe of
*T. transversa* might be mediated by *fz1/2/7*. Therefore, the fact that *wnt5*
expression correlates with the growing/elongating mantle lobe, and is expressed
at the growing tips, suggests that convergent extension might be the
developmental mechanism involved and the basis on how the mantle is formed.

<!--TODO: Maybe check the wnt1 receptor and refer to the apical-mantle above.-->

The posteriormost Frizzled-expressing territory in *T. transversa* is a
distinct band of *fz9/10* immediately posterior to the mantle lobe. Wnt ligands
here are expressed in partially overlapping domains, but covering subregions.
*wnt1* for instance occupies the ventral and lateral portion, while *wnt6* is
on the dorsal side. Here again we observe the same arrangement of *wnt1* and
*wnt7* as present in the apical-mantle lobe boundary. We observe a similar or
different arrangement in other animals [@refs]. The fact that seem to coincide
with the *fz9/10* territories during *T. transversa* gastrulation is rather
interesting and suggest they could be interacting with this particular
receptor.

<!--TODO: Check interaction between wnt1 and wnt7 and fz9/10.-->

<!--TODO: Find out if any of these posterior genes have any role elsewhere.-->

Finally, at the posterior tip of the T. transversa embryo there is a
Frizzled-free zone where the ligands *wnt5*, *wnt6* and *wnt11* are expressed.
One possibility is that these genes are interacting with other co-receptors
[@refs]. It seems likely they are not signaling via the cell fate pathway
mediated by β-catenin (canonical). Such region might not respond to the
overactivation of the Wnt pathway, and retain the terminal identity, as in the
wnt-independent posterior tip region described in @Darras. 

<!--TODO: Look into wnt5, wnt6, and wnt11 posterior terminal expression.-->

Overall, the specific ligand-receptor combinations expressed along the
anteroposterior axis of *T. transversa* coincides with the major body regions
and their subregions. This suggests that the different ligand-receptor context
can be involved in setting up the identities or differentially patterning the
tissue fates of the different body regions of the brachiopod *T. transversa*.

<!--TODO: Discuss mesodermal expression?-->
<!--TODO: Discuss axin, dishevelled, bcat?-->
<!--TODO: How does Wnt signaling intersects with Hox expression?-->

## Wnt ligands under different receptor-contexts

*wnt1*, *wnt5* and *wnt7* are expressed in multiple body regions, and
different receptor contexts. *wnt5* is expressed in the apical lobe, in the
mantle lobe and in the terminal portion of the pedicle lobe. Therefore, the
expression of the ligand overlaps with the expression of the receptors
*fz1/2/7* and *fz4* in the apical lobe, *fz1/2/7* in the mantle lobe and no
Frizzled in the pedicle lobe. These different receptor contexts coincide with
different morphological outcomes of these body regions.

The posterior region of the apical lobe where *wnt5* and other Wnts are
expressed, corresponds to a region that undergoes intense cell proliferation
forming an enlarged appearance in the late larva. The region expresses *fz4*
which is associated with the activation of the *β-catenin* cascade
(canonical Wnt pathway) [@refs], and therefore could be mediating the cell
proliferation in the apical lobe.

The mantle lobe, in contrast, *wnt5* only overlaps by *fz1/2/7*. The region
undergoes dramatic morphogenetic changes during development, where the tissues
grow and elongate forming an extended mantle in the late larva. Interestingly,
*wnt5* regulates tissue elongation by controlling convergent extension in other
animals [see refs in @Loh2016-pz]. These morphogenetic changes might depend
upon the activation of the cell polarity cascade (noncanonical Wnt pathway) and
the signaling cascade in the mantle lobe of *T. transversa* might be mediated
by *fz1/2/7*. We thus speculate that the developmental mechanism involved in
the elongation of the mantle lobe is convergent extension.

<!--TODO: Discuss how wnt1 antagonizes wnt5, see Torres1996-hs-->

<!--TODO: Get examples of Wnt activity without Frizzled receptors.-->

The activity of Wnt ligands without Frizzled receptors has been described in
[@refs]. That might be the case for the terminal wnt5 domain in the pedicle
lobe.

Another example is *wnt1* and *wnt7* where in the region containing *fz1/2/7*
and *fz9/10* the stripes coincide with the deep epithelial invagination of the
apical-mantle boundary, while in the region containing only *fz9/10* the
morphological features are different. wnt7 has been described to be involved in
[@refs]. In addition, the apical-mantle boundary is abutted by *wnt1*
(anterior) and *engrailed* (posterior) stripes [@Vellutini2016-lq]. This
arrangement is highly conserved in arthropod trunk segmentation and has a role
in setting up segment polarity [@refs]. In Drosophila and others, it is known
or suggested that the receptor mediating wnt1 signaling is fzX [@refs]. In *T.
transversa* this seems to be the same or different because the Frizzled
receptors expressed in the apical-mantle boundary are fz1/2/7 and fz9/10.

These observations suggest that the receptor context, and possibly the presence
of other ligands, might play a role in determining the signaling being
activated. And therefore, be important for the differentiation of brachiopod
larval body regions.

## Conserved ligand--receptor subregions between spiralian and deuterostomes

<!--TODO: Remove these first paragraphs and keep only hemichordate?-->

If the ligand-receptor context plays a role in specifying different brachiopod
body regions, we wondered how conserved these transcriptional subregions are
among other animals. Unfortunately, the spatial Wnt expression data in the
brachiopod *L. anatina* is limited, so we first compare to other spiralians.

In the annelid *Platynereis dumerilii* we find several correspondent, putative
homologous Wnt expression domains with *T. transversa*, despite the great
differences in embryo architecture -- the annelid has fewer cells than the
brachiopod at the moment of gastrulation [@Cho2010-eb; @Janssen2010-dc]. For
instance, the expression of Frizzled receptors in *P. dumerilii* are mostly
similar to what we find in the brachiopod, following the same anteroposterior
arrangement [@Bastin2015-oz]. We can also identify Wnt ligands with comparable
expression domains. For example, the posterior expression of *wnt1* and the
anterior domain of *wnt2* are similar [@Prudhomme2003-lu; @Dray2010-op;
@Janssen2010-dc; @Pruitt2014-oy]. However, the spatial arrangement of some Wnt
ligands are clearly not conserved between *T. transversa* and *P. dumerilii*.
Data is still lacking from other spiralians specially from early stages that
have not been described. 

<!--TODO: Make a proper comparison for the domains.-->

<!--TODO: Which other spiralian can I compare to?-->

<!--TODO: Deuterostome (hemichordate) comparison, common domains.-->

The expression of Wnt genes in Ecdysozoa reveals that some domains are
comparable [@refs]. However, in the priapulid Priapulus caudatus most Wnt genes
are expressed at the posterior region of the embryo [@Hogvall2019-wq].
Expression of Wnt ligands in Drosophila and other arthropods show similarities.
However, the Frizzled domains do not entirely match what we find in *T.
transversa* [@refs]. *D. melanogaster* fz2 expression, orthologous to *T.
transversa* fz5/8 is not expressed apically, but in the trunk, similar to T.
transversa fz9/10 [@refs]. In addition, arthropods lost many Wnt ligands in
specific lineages complicating the identification of common or divergent
ligand-receptor contexts [@refs].

<!--TODO: Research better and find some examples of comparable domains.-->

<!--TODO: Are the Frizzled domains conserved, at least?-->

In the deuterostomes the Wnt signaling landscape of hemichordates has been
analyzed in detail [@Darras2018-gm], and thus allow for a more complete
comparison to the brachiopod findings. The spatial arrangement of Frizzled
domains is similar between *T. transversa* and *Saccoglossus kowalevskii*
(@fig:comparative). In a closer comparison, we can also identify that many Wnt
ligands are also expressed in similar positions along the anteroposterior axis
of the two species (@fig:comparative). However, there is also differences.

By compiling the patterns, we can identify a common set of ligand-receptor
territories that are similar between brachiopods and hemichordates
(@fig:comparative). These are the anterior *fz1/2/7*+*fz4* domains with *wnt2*,
*wnt5* and *wnt8*, the head-trunk boundary expressing fz1/2/7+fz9/10 and
*wnt1*, the anterior portion of the trunk expressing *wnt16*, the posterior
trunk expressing *fz9/10* and *wnt10*, and the Frizzled-free posterior zone
expressing *wnt6*. These combinations can be merely coincidental, and more data
is needed to check if these unique regions occur in other bilaterians.

<!--TODO: Fetch relevant data for comparison.-->

Comparing to other deuterostomes such as echinoderms [@Robert2014-xj],
ascidians, amphioxus and vertebrates shows that Frizzled territories are
conserved [@refs]. We can also identify some ligand-receptor context that
appear often [@refs] and some others not.

Finally, when we put into a metazoan context, some recapitulate the expression
in cnidarians where arrangement of the ancestral Wnt ligands is not that
different [@refs]. However, the data on Frizzled is lacking. But the global
picture of Wnt signaling landscape is similar to hemichordates.

Overall, the emerging picture is that these molecular territories can
follow the same arrangement, the morphological boundaries demarcate body
regions can differ from group to group.

<!--For example, that is the case for the apical-mantle boundary and the collar-trunk boundary as the head-trunk boundary. These boundaries occur at the interface of different molecular contexts.-->

<!--TODO: Collect more convincing references.-->

## Role of ligand--receptor changes to bilaterian evolution

A common consensus is that axial patterning genes of bilaterians are conserved
and inherited from the last common ancestor [@refs]. The similar arrangement of
our Wnt signaling patterning data support this view. The discovery of multiple
wnt genes, their crucial developmental roles and their complex expression
domains has raised ideas about a wnt code [@Guder2006-fu]. An idea that
different Wnts can pattern and specify different body regions according to
their combinatorial territories.

First, the finding that wnt ligands are not long- but short-ranged signaling
activity [@Loh2016-pz]. The consequence is that wnt function is more local and
allow for more complex patterning outcomes. Second, the idea that is not a
specific wnt ligand that mediate specific signaling, but that the receptor
context dictates the Wnt signaling outcome [@Loh2016-pz]. This implies that the
same ligand can trigger different outcomes based on the receptor it binds to.
This put a greater importance when studying Wnt signaling on analyzing the
receptor context in conjunction to ligands. Therefore, accumulated evidence
reveals that the Wnt system is much more imbricated modular and multifunctional
than the Hox patterning.

<!--TODO: Develop better the final argument.-->

In that case, evolution might have tinkered with downstream regulators, genes
activated by the Wnt signaling pathway to generate different morphological
outcomes during anteroposterior patterning, as we observed in the examples
above. Therefore, the intricacy, pleiotropy, redundancy, modularity of the Wnt
signaling system might have paved the way and been crucial for the evolution of
bilaterian diversity.

The complex arrangements of wnt expression domains along the anteroposterior of
different animals suggests they might play a key patterning role. Since it is
at this point that major morphological features are specified, understanding
the roles of Wnts is crucial to understand the evolution of animal morphology.
However, given the recent developments on wnt signaling, we will not manage by
only looking at ligands. Thus it is crucial to consider the unique combination
of ligand-receptor frizzled and other receptors in future comparative studies.

<!--TODO:

    - @Marlow2014 conservation of wnt antagonist domains
    - @Darras2011 wnt siganling
    - @Van_Amerongen2009-wk review

    -->

# Methods

## Sample collection

Adult *T. transversa* (Sowerby, 1846) specimens were collected by dredging the
rocky seabeds around Friday Harbor, San Juan Islands, USA. They were kept in a
tank with running seawater at the Friday Harbor Laboratories (University of
Washington). To obtain embryos, we dissected the gonads of ripe individuals and
fertilized the gametes *in vitro* as previously described [@Reed1987-sd;
@Freeman1993-zt]. The embryos developed in glass bowls with filtered seawater,
partially immersed in running seawater for a stable culturing temperature
(around 7.6°C in January). Samples for *in situs* were fixed in 4% formaldehyde
at room temperature for 1h, washed thoroughly in 1x PBS with 0.1% Tween-20, and
stored in 100% methanol at -20°C. Samples for RNA-seq were preserved directly
in RNAlater at representative developmental stages (@tbl:samples). We collected
two biological replicates, each containing the eggs of a single female
fertilized with the sperm of three different males.

## RNA sequencing and analyses

We extracted the total RNA of individual RNA-seq samples using Trizol. The 28
samples were randomized on four lanes of a Illumina HighSeq 2000 at the EMBL
Genomic Core Facilities (GENECORE). We sequenced 24±5 million 50bp single-end
reads on average across samples. We quantified the transcript abundances using
Kallisto v0.46.0 [@Bray2016-lm] to pseudoaligned the reads to a reference
transcriptome of *T. transversa* (assembled from
[SRX1307070](https://www.ncbi.nlm.nih.gov/sra/SRX1307070[accn])). The
percentage of uniquely mapped reads was high for all samples (85.6±1.7%). We
used DESeq2 [@Love2014-hs] to estimate the library size factors and the data
dispersion, and to homogenize the variance across expression ranks
(variance-stabilizing transformation) for sample clustering and visualization.
We used pheatmap [@Kolde_undated-gt] and ggplot2 [@Wickham2016-rz] packages to
create plots in R [@Development_Core_Team2005-ki] running RStudio Desktop
[@RStudio_Team2020-wr].

## Gene orthology

We used the sequences of known Wnt signaling components to identify similar
sequences in the reference transcriptome of *T. transversa* using BLAST+
[@Camacho2009-jo]. Our searches yield several candidates. To resolve their
orthology, we aligned the obtained protein sequences of *T. transversa* with
well-annotated genes from other metazoans using MAFFT 7.310 [@Katoh2013-mz].
Using GBlocks 0.91b [@Talavera2007-fl], we removed non-informative sections of
the alignment and inspected the multiple sequence alignment visually using
UGENE [@Okonechnikov2012-mr]. We used the blocked alignments as input to run a
maximum likelihood phylogenetic analysis using automatic model recognition and
rapid bootstrap options of RAxML 8.2.12 [@Stamatakis2014-bm]. We rendered the
resulting trees using the Interactive Tree Of Life web application
[@Letunic2016-pn]. The gene orthology analyses and source files are available
in the repository [TODO].

## Cloning and *in situ* hybridization

We designed gene-specific primer pairs for the identified *T. transversa* Wnt
signaling components using Primer3 [@Untergasser2012-se]. We targeted the
coding sequence to obtain product sizes between 800--1200 bp. We synthesized
cDNA using the SMARTer RACE cDNA Amplification kit (Clontech), and amplified
the fragments by PCR, and cloned them into pGEM-T Easy Vector. We amplified
antisense sequences using T7 or SP6 polymerase, and synthesized DIG-labeled
riboprobes with MEGAscript kit (Ambion). We then followed established protocols
in *T. transversa* for single colorimetric *in situ* hybridization
[@Hejnol2008-pk; @Santagata2012-he], and double fluorescent *in situ*
hybridization [@Vellutini2016-lq; @Martin-Duran2016-qa]. The primer pairs are
available in the repository [TODO] and the cloned sequences were deposited in
the GenBank (XXX--XXX).

## Microscopy and image processing

We mounted the embryos between a glass slide and a coverslip, supported by clay
feet, in 70% glycerol in PBS. We used a Zeiss AxioCam HRc mounted on a Zeiss
Axioscope A1 to image the samples under differential interference contrast
technique (Nomarski). For fluorescent samples, we scanned volumetric stacks in
a Confocal Leica TCS SP5 microscope and generated maximum intensity projections
using Fiji [@Schindelin2012-di]. We adjusted the distribution of intensity
levels to improve the contrast using Fiji or GIMP. We made illustrations and
assembled figure plates using Inkscape.

# Acknowledgements

We thank the Friday Harbor Laboratories boat crew for collecting the
brachiopods. Yale Passamaneck for helping with the spawnings. Katrine Worsaae
for the initial cloning batch. Members of the Hejnol Lab for the helpful
discussions. The study was funded by the Sars Centre core budget and by The
European Research Council Community’s Framework Program Horizon 2020
(2014–2020) ERC grant agreement 648861. The animal collections were supported
by Meltzer Research Fund grant. BCV was supported by an EMBO Long-Term
Fellowship (ALTF 74-2018).

# References

::: {#refs}
:::

\clearpage

\newpage

# Figures

![Orthology assignment of *Terebratalia transversa* Wnt ligands. Maximum
likelihood phylogenetic analysis using the amino acid sequences of known
metazoan Wnt genes. Color-coding represents Wnt subfamilies. Numbers show the
bootstrap branch support values. *T. transversa* orthologs are in bold and
outlined by a box.](figures/Fig1.jpg){#fig:wnt-tree width=100%}

![Expression of Wnt signaling components throughout *Terebratalia transversa*
development. Heatmap shows the log normalized counts averaged between
replicates from stage-specific RNA-seq data. The stages sampled for *in situ*
hybridization are highlighted in magenta.](figures/Fig2.jpg){#fig:profiling
width=100%}

![Whole-mount *in situ* hybridization of Wnt ligands in *Terebratalia
transversa* (*wnt1*, *wnt1b*, *wnt2*, *wnt4*, *wnt5*, *wnt6* and *wnt7*). Black
arrowheads indicate the apical--mantle boundary. White arrowheads demarcate the
mantle--pedicle boundary. The panels for *wnt1* were originally published in
@Vellutini2016-lq under a Creative Commons Attribution License and reprinted
here for completion.](figures/Fig3.jpg){#fig:ligands1 width=100%}

![Whole-mount *in situ* hybridization of Wnt ligands in
*Terebratalia transversa* (*wnt8*, *wnt9*, *wnt10*, *wnt11*, *wnt16* and
*wntA*). Black arrowheads indicate the apical--mantle boundary. White
arrowheads demarcate the mantle--pedicle
boundary.](figures/Fig4.jpg){#fig:ligands2 width=100%}

![Whole-mount double-fluorescent *in situ* hybridization of Wnt ligands in
*Terebratalia transversa*. (A--D) Expression of *wnt1* (magenta) and *wnt5*
(green) for early gastrula (A), late gastrula (B) and early larva (C) in
ventral views, and for early larva in dorsal view (D). (E--F) Expression of
*wnt5* (green) and *fz5/8* (magenta) for early gastrula (E) and late gastrula
(F) in ventral views. (G--H) Expression of *wnt7* (green) *engrailed* (*en*)
(magenta) in ventral (G) and dorsal (H) views. (I--L) Expression of *wnt1*
(magenta) and *wnt8* (green) for early gastrula (I) and early larva (J--L)
in ventral view (J), optical section through mid-body (K) and dorsal view
(L).](figures/Fig5.jpg){#fig:doubles width=100%}

![Whole-mount *in situ* hybridization of Frizzled receptors in
*Terebratalia transversa*. Black arrowheads indicate the apical--mantle
boundary. White arrowheads demarcate the mantle--pedicle
boundary.](figures/Fig6.jpg){#fig:receptors width=100%}

![Whole-mount *in situ* hybridization of Wnt pathway antagonists in
*Terebratalia transversa*. Black arrowheads indicate the apical--mantle
boundary. White arrowheads demarcate the mantle--pedicle
boundary.](figures/Fig7.jpg){#fig:antagonists width=100%}

![Whole-mount *in situ* hybridization of intracellular components of the Wnt
pathway in *Terebratalia transversa*. Black arrowheads indicate the
apical--mantle boundary. White arrowheads demarcate the mantle--pedicle
boundary.](figures/Fig8.jpg){#fig:intracellular width=100%}

![Summary of Wnt signaling landscape in *Terebratalia transversa*. The
schematic drawings show the ectodermal expression domains on the ventral
region of a late gastrula stage.](figures/Fig9.jpg){#fig:summary width=100%}

![Comparative Wnt signaling ligand--receptor landscape between the brachiopod
*Terebratalia transversa* and the hemichordate *Saccoglossus kowalevskii*.
Broad receptor territories are essentially the same along the anteroposterior
axis. There are common ligand--receptor combinations between the two animals
forming unique transcriptional regions in the embryo that corresponds to
differentiated body regions.](figures/Fig10.jpg){#fig:comparative width=100%}

\newpage

# Tables

Table: Maternal load of Wnt ligands and Frizzled receptors in metazoan eggs.
{#tbl:maternal}

| Species                         | Wnt ligands                                         | Frizzled receptors                                       |
|---------------------------------|-----------------------------------------------------|----------------------------------------------------------|
| *Terebratalia transversa*       | *wnt4* and *wntA* (this study)                      | *fz1/2/7*, *fz5/8* and *fz4* (this study)                |
| *Lingula anatina*               | *wntA* and *wnt8* [@Luo2015-lm; @Luo2018-lj]        | *fz1/2/7*, *fz4* and *fz9/10* [@Luo2015-lm; @Luo2018-lj] |
| *Platynereis dumerilii*         | No maternal Wnts [@Pruitt2014-oy]                   | *fz1/2/7* [@Bastin2015-oz]                               |
| *Priapulus caudatus*            | *wnt2*, *wnt4*, *wnt5* and *wnt8* [@Hogvall2019-wq] | -                                                        |
| *Paracentrotus lividus*         | *wnt7* [@Robert2014-xj]                             | *fz1/2/7* and *fz5/8* [@Robert2014-xj]                   |
| *Strongylocentrotus purpuratus* | *wnt6*, *wnt7* and *wnt16* [@Croce2011-ae]          | -                                                        |
| *Saccoglossus kowalevskii*      | *wnt4* and *wnt9* [@Darras2018-gm]                  | *fz1/2/7* and *fz5/8* [@Darras2018-gm]                   |
| *Xenopus laevis*                | *wnt5*, *wnt8* and *wnt11* [@Tao2005-qc]            | -                                                        |
| *Danio rerio*                   | *wnt8* [@Nasevicius1998-bm]                         | -                                                        |
| *Clytia hemisphaerica*          | *wnt3* [@Momose2008-va]                             | *fz1* and *fz3* [@Momose2008-va]                         |

\newpage

Table: Wnt signaling subregions in the brachiopod *T. transversa*.
{#tbl:landscape}

| Region                    | Antagonists     | Ligands                                            | Receptors             |
| ------------------------- | --------------- | -------------------------------------------------- | --------------------- |
| Anterior tip              | *dkk*, *sfrp1/2/5*   | (*wnt1b*)                                                  | *fz1/2/7*, *fz5/8*    |
| Apical lobe               | -               | *wntA*, *wnt8*, *wnt2*, *wnt5*                     | *fz1/2/7*, *fz4*      |
| Apical--mantle boundary   | -               | *wnt1*, *wnt7*, *wnt16*                            | *fz1/2/7*, *fz9/10*   |
| Mantle lobe               | -               | *wnt5*, *wnt16*                                    | *fz1/2/7*             |
| Pedicle lobe              | -               | *wnt1*, *wnt9*, *wnt11*, *wnt4*, *wnt10*, *wnt7*   | *fz9/10*              |
| Posterior tip             | -               | *wnt6*, *wnt11*, *wnt5*                            | -                     |

\newpage

Table: Sampled developmental stages for the stage-specific transcriptome of the
brachiopod *T. transversa*. {#tbl:samples}

| Stage ID | Time post fertilization | Stage description                    |
| -------- | ----------------------- | ------------------------------------ |
| S01      | 0h                      | oocytes                              |
| S02      | 8h                      | mid blastula                         |
| S03      | 19h                     | late blastula                        |
| S04      | 24h                     | moving late blastula                 |
| S05      | 26h                     | early gastrula                       |
| S06      | 37h                     | asymmetric gastrula                  |
| S07      | 51h                     | bilateral gastrula                   |
| S08      | 59h                     | bilobed                              |
| S09      | 68h                     | trilobed                             |
| S10      | 82h                     | early larva (first chaetae visible)  |
| S11      | 98h                     | late larva (long chaetae, eye spots) |
| S12      | 131h                    | competent larva                      |
| S13      | 1d                      | juvenile                             |
| S14      | 2d                      | juvenile                             |

\newpage

# Supplementary material

## Figures

![Analysis of *Terebratalia transversa* *wnt1* paralogs. Maximum likelihood
phylogenetic analysis of a multiple sequence alignment of amino acid sequences
for *wnt1* of diverse metazoans. Numbers show the bootstrap branch support
values.](figures/FigS1.jpg){#fig:wnt1-tree tag=S1 width=100%}

![Whole-mount colorimetric *in situ* hybridization of Wnt ligands in
*Terebratalia transversa*.](figures/FigS2.jpg){#fig:s-ligands tag=S2
width=100%}

![Whole-mount fluorescent double-fluorescent *in situ* hybridization of Wnt
ligands in *Terebratalia transversa*.](figures/FigS3.jpg){#fig:s-doubles tag=S3
width=100%}

![Orthology assignment of *Terebratalia transversa* Frizzled receptors. Maximum
likelihood phylogenetic analysis using the amino acid sequences of known
metazoan Frizzled genes. Color-coding represents Frizzled subfamilies. Numbers
show the bootstrap branch support values. *T. transversa* orthologs are in
bold.](figures/FigS4.jpg){#fig:fzd-tree tag=S4 width=100%}

![Whole-mount colorimetric *in situ* hybridization of Frizzled receptors in
*Terebratalia transversa*.](figures/FigS5.jpg){#fig:s-receptors tag=S5
width=100%}

![Orthology assignment of *Terebratalia transversa* Wnt antagonists secreted
Frizzled-related proteins. Phylogenetic analysis of the amino acid sequences of
known metazoan *sfrp* genes.](figures/FigS6.jpg){#fig:sfrp-tree tag=S6
width=100%}

![Whole-mount colorimetric *in situ* hybridization of Wnt antagonists in
*Terebratalia transversa*.](figures/FigS7.jpg){#fig:s-antagonists tag=S7
width=100%}

![Orthology assignment of *Terebratalia transversa* Wnt antagonist *dkk*.
Phylogenetic analysis of the amino acid sequences of known metazoan *dkk*
genes.](figures/FigS8.jpg){#fig:dkk-tree tag=S8 width=100%}

![Orthology assignment of *Terebratalia transversa* Wnt antagonist *wif*.
Phylogenetic analysis of the amino acid sequences of known metazoan *wif*
genes.](figures/FigS9.jpg){#fig:wif-tree tag=S9 width=100%}

![Orthology assignment of *Terebratalia transversa* Wnt intracellular component
*dsh* and *axin*. Phylogenetic analysis of the amino acid sequences of known
metazoan *dsh* and *axin* genes.](figures/FigS10.jpg){#fig:dsh-tree tag=S10
width=100%}

![Orthology assignment of *Terebratalia transversa* Wnt intracellular component
*bcat*. Phylogenetic analysis of the amino acid sequences of known metazoan
*bcat* genes.](figures/FigS11.jpg){#fig:bcat-tree tag=S11 width=100%}

![Expression of intracellular Wnt signaling components throughout *Terebratalia
transversa* development. Heatmap shows the log normalized counts from
stage-specific RNA-seq data.](figures/FigS12.jpg){#fig:s-profiling tag=S12
width=100%}

![Whole-mount colorimetric *in situ* hybridization of Wnt intracellular
components in *Terebratalia
transversa*.](figures/FigS13.jpg){#fig:s-intracellular tag=S13 width=100%}

![Orthology assignment of *Terebratalia transversa* cell polarity component
*diego* (*diversin* or *ankrd6*). Phylogenetic analysis of the amino acid sequences of
known metazoan *diego* genes.](figures/FigS14.jpg){#fig:dgo-tree tag=S14
width=100%}

![Whole-mount colorimetric *in situ* hybridization of the cell polarity Wnt
signaling pathway in *Terebratalia
transversa*.](figures/FigS15.jpg){#fig:s-polarity tag=S15 width=100%}

![Orthology assignment of *Terebratalia transversa* cell polarity component
*prickle*. Phylogenetic analysis of the amino acid sequences of known metazoan
*prickle* genes.](figures/FigS16.jpg){#fig:pk-tree tag=S16 width=100%}

![Orthology assignment of *Terebratalia transversa* cell polarity component
*flamingo*. Phylogenetic analysis of the amino acid sequences of known metazoan
*flamingo* genes.](figures/FigS17.jpg){#fig:fmi-tree tag=S17 width=100%}

![Orthology assignment of *Terebratalia transversa* cell polarity component
*strabismus*. Phylogenetic analysis of the amino acid sequences of known
metazoan *strabismus* genes.](figures/FigS18.jpg){#fig:stbm-tree tag=S18
width=100%}

![Orthology assignment of *Terebratalia transversa* cell polarity component
*jnk*. Phylogenetic analysis of the amino acid sequences of known metazoan
*jnk* genes.](figures/FigS19.jpg){#fig:jnk-tree tag=S19 width=100%}

